### PR TITLE
navbar title fixed #2150

### LINKED ISSUE #2150: Hover on navbar Links shows "main-navigation", "user-menu" and the drop down arrow shows "triangle"
m5
m1
m2
m3
m4

Also hovering on the P5.js logo at the top corner shows "main-navigation"
If its a valid issue can I work on it?

### PR DESCRIPTION
**Pull request by Aryan1982**
Fixes issue #2150

Changes:
Before:
m1
m2

After:
n1

n2

### CODE REVIEW

**Review by lindapaiste — DISMISSED** [review]

**Comment by Aryan1982** [comment]
Annotation 2023-03-10 111748


This check is failing, am I missing something?

**Comment by 3ru** [comment]
@Aryan1982 
It is failing on the snapshot test of jest. Please update it below and commit again.

```bash
jest --updateSnapshot
```

**Comment by Aryan1982** [comment]
@3ru 
Done.

File: package.json
Comment (3ru) [inline package.json]: These should not be added.

**Review by 3ru — CHANGES_REQUESTED** [review]
Do not include `package.json` and `package-lock.json` in this PR, but `client/components/__snapshots__/Nav.unit.test.jsx.snap` and `client/components/Nav.jsx`. Only.

You can edit previous commits with the git command.

File: package.json
Comment (Aryan1982) [reply package.json]: @3ru 
I tried git commands but I did not managed to edit my previous commit with package.json, and ended up commiting one more change. how do I unstage package.json from [b9e4bef] this commit , and is it possible to delete the unwanted commits after that?

Comment (3ru) [reply package.json]: @Aryan1982 
The maintainer can use squash, etc., to summarize when merging, so there is no problem with commits being added.

For now, only `client/components/Nav.jsx` has been added, and it will fail the test again.

- update
`jest --updateSnapshot`
- add
`git add client/components/__snapshots__/Nav.unit.test.jsx.snap`

After that, commit and push.

---
It is also a good idea to try `npm run test` in your own environment to see if the test fails.

If the test passes as follows, you will not fail in CI.

```bash
Test Suites: 15 passed, 15 total
Tests:       109 passed, 109 total
Snapshots:   2 passed, 2 total
```

Comment (Aryan1982) [reply package.json]: @3ru 
Thank you! I have updated the snapshot, and going to add and commit changes

Comment (Aryan1982) [reply package.json]: @3ru 
got the following
` PASS   client  client/components/Nav.unit.test.jsx (10.815 s)
 PASS   client  client/modules/IDE/components/FileNode.unit.test.jsx (9.3 s)
 PASS   server  server/controllers/user.controller/__tests__/apiKey.test.js (5.711 s)
 PASS   server  server/controllers/project.controller/__test__/deleteProject.test.js (7.267 s)
 PASS   server  server/controllers/project.controller/__test__/getProjectsForUser.test.js (9.191 s)
 PASS   server  server/domain-objects/__test__/Project.test.js (7.213 s)
 PASS   server  server/models/__test__/project.test.js
 PASS   client  client/modules/IDE/actions/projects.unit.test.js (5.441 s)
 PASS   client  client/utils/getConfig.test.js

Test Suites: 1 failed, 14 passed, 15 total
Tests:       1 failed, 108 passed, 109 total
Snapshots:   2 passed, 2 total`

Comment (Aryan1982) [reply package.json]: snapshots was failing before, after updating..its fine now, 
what about the other tests?

Comment (3ru) [reply package.json]: @Aryan1982 
The `Nav.unit.test.jsx` test seems to pass, so I think this PR will be OK if you push the snapshot and commit.

That test is failing is another issue.

**Comment by Aryan1982** [comment]
@3ru 
This doesn't feel quiet right.. have I done any mistake?

**Comment by 3ru** [comment]
@Aryan1982 
Yes, I did not expect it to be so complicated.
Could you please invite me to [your fork repository](https://github.com/Aryan1982/p5.js-web-editor)?

**Comment by Aryan1982** [comment]
Thank you @3ru  , could you tell me what I was doing wrong?

**Comment by 3ru** [comment]
Yes!
<details>
<summary>What was the problem</summary>

The two major problems so far have been
1. The component was changed, but the corresponding test snapshot was not added.
2. Library updates (changes to package.json) were included.

Perhaps for some reason, you updated `jest` and then did a `git add . `, etc. and you committed everything, but the PR needs to be the minimum change necessary to fix the problem.
For example, in this case, you only need to update the Nav component and the test file for that component, so you only need to commit those two files.

</details>

<details>
<summary>Fixes.</summary>

Here are the flow and commands I modified.

1. Rewind to the very first commit by @Aryan1982
    ```zsh
    git reset --hard c544c68c0571dd1acbb99e282e2167710eed1cb5
    ```

2. Force overwrite the rewind to the remote repository
    ```zsh
    git push -f origin develop
    ```

3. Update snapshot of just
    ```zsh
    jest --updateSnapshot
    ```

4. Add only the snapshot to the commit & push
    ```zsh
    git add client/components/__snapshots__/Nav.unit.test.jsx.snap
    
    git commit -m "update nav unit snapshot"
    git push origin develop
    ```

</details>


It has been a long journey because of my inability to explain.:pray: 
Now let's wait for @raclim's review.

**Review by 3ru — APPROVED** [review]
lgtm

**Comment by Aryan1982** [comment]
Thank you @3ru  for helping me with my first contribution to a open source project. Do you have any socials?

**Review by raclim — APPROVED** [review]

**Pull request merged**
